.gitignore

```diff
@@ -4,6 +4,7 @@
 /.fleet/
 /cmd/*/__debug_bin
 /generate_*_types
+/print_*_types
```

Comment:
why are the two here? was it just rename or are both relevant?

Reply:
That is just how GoLand stores binaries, I could not figure out how to tell it to store them elsewhere.